package.json

```diff
@@ -148,7 +148,7 @@
     },
     "lint-staged": {
         "*.{js,ts,tsx,json,yml,css,scss}": "prettier --write",
-        "*.{js,ts,tsx}": "eslint",
+        "*.{js,ts,tsx}": "eslint --fix",
```

Comment:
Not 100% sure about this? Could this potentially lead to some unexpected changes? I do think that most of the time the fixes are quite straightforward